specs-go/config.go

```diff
@@ -33,6 +34,14 @@ type DeviceNode struct {
 	Permissions   []string `json:"permissions,omitempty"`
 }
 
+// User represents a user the container process runs as
+type User struct {
+	UID            uint32   `json:"uid"`
+	GID            uint32   `json:"gid"`
+	Umask          uint32   `json:"umask,omitempty"`
+	AdditionalGids []uint32 `json:"additionalgids,omitempty"`
+}
```

Comment:
What do you think about using pointers for unset on omit empty...

Reply:
that makes sense. Will do.